src/main/java/technology/tabula/TextChunk.java

```diff
@@ -62,7 +152,7 @@ public String getText() {
         for (TextElement te: this.textElements) {
             sb.append(te.getText());
         }
-        return sb.toString();
+        return Normalizer.normalize(sb.toString(), Normalizer.Form.NFKD);
```

Comment:
`TextChunk.getText` is the last method that gets called when emitting a table, so IMO that's the best place to normalize text.